checkov/sca_package/scanner.py

```diff
@@ -2,35 +2,55 @@
 import json
 import logging
 import os
+import time
 from collections.abc import Iterable, Sequence
 from pathlib import Path
 from typing import Tuple, Dict, Any
 
 from aiomultiprocess import Pool
 
+from checkov.common.bridgecrew.platform_key import bridgecrew_dir
 from checkov.common.bridgecrew.vulnerability_scanning.integrations.package_scanning import package_scanning_integration
 
+
 TWISTCLI_FILE_NAME = 'twistcli'
+SEC_IN_WEEK = 604800
+
+
+def get_expiration_time_in_sec():
+    return int(os.getenv("EXPIRATION_TIME_IN_SEC", SEC_IN_WEEK))
```

Comment:
@NoaAzoulay 2 things:
1. Please prefix all env vars with `CHECKOV_`
2. do we need to wrap this as a function? Can it be written in-line in the caller function?